app/elm/Main.elm

```diff
@@ -26,4 +26,6 @@ initialModel : Model
 initialModel =
     { postcode = ""
     , categories = allCategories
+    , date = ""
```

Comment:
What is this date for? Is it for filtering? It seems a bit ambiguous (might be worth naming it something different)